src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -107,16 +109,27 @@ const UserProfile = ({
         <ToggleSwitch
           value={isAnonymous}
           onValueChange={() => setTypeUser(!isAnonymous)}
-          labelLeft="Anonymity"
+          labelLeft="Incognito"
           backgroundActive={COLORS.lightgrey}
           backgroundInactive={COLORS.lightgrey}
           circleInActiveColor={COLORS.signed_primary}
           inactiveTextColor={COLORS.signed_primary}
           styleLabelLeft={styles.switch}
+          isDisabled={isToggleDisabled}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The label of the ToggleSwitch has been changed from "Anonymity" to "Incognito". Also, a new prop `isDisabled` has been added to the ToggleSwitch component. Make sure that the ToggleSwitch component supports this prop and behaves as expected when it's set to true.
```diff
-          labelLeft="Anonymity"
+          labelLeft="Incognito"
+          isDisabled={isToggleDisabled}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->